lib/reducers/term-groups.js

```diff
@@ -192,7 +192,10 @@ const reducer = (state = initialState, action) => {
         return setActiveGroup(state, action);
       }
 
-      const uid = uuid.v4();
+      let uid = uuid.v4();
```

Comment:
Preserving `const` is maybe better here: `const uid = action.termGroupUid || uuid.v4()`